packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/ChatbotHeaderDrawer.tsx

```diff
@@ -31,12 +31,25 @@ const initialConversations: { [key: string]: Conversation[] } = {
   ]
 };
 
+const ERROR = {
+  bodyText:
+    'Something went wrong while loading your chat history. Please check your connection and try again or refresh this page.',
```

Comment:
```suggestion
    'To try again, check your connection and refresh this page. If the issue persists, [contact the support team]().',
```
You can also leave off this second sentence! Just thinking about including this sort of language based on user testing so far